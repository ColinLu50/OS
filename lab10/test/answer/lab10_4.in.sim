43416
1501
1023
837
1019
825
