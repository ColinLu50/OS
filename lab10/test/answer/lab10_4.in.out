FCFS
193 -> 21 -> 84 -> 313 -> 202 -> 254 -> 605 -> 145 -> 253 -> 754 -> 96 -> 2 -> 317 -> 345 -> 245 -> 671 -> 125 -> 54 -> 809 -> 399 -> 362 -> 47 -> 759 -> 421 -> 525 -> 34 -> 753 -> 87 -> 623 -> 512 -> 256 -> 469 -> 563 -> 495 -> 616 -> 749 -> 177 -> 48 -> 261 -> 777 -> 33 -> 488 -> 9 -> 486 -> 650 -> 338 -> 727 -> 124 -> 373 -> 140 -> 50 -> 109 -> 354 -> 513 -> 713 -> 291 -> 680 -> 280 -> 583 -> 186 -> 604 -> 339 -> 265 -> 588 -> 321 -> 371 -> 371 -> 827 -> 451 -> 236 -> 684 -> 216 -> 449 -> 35 -> 203 -> 191 -> 327 -> 480 -> 653 -> 266 -> 585 -> 279 -> 733 -> 88 -> 607 -> 82 -> 411 -> 531 -> 666 -> 140 -> 80 -> 441 -> 116 -> 307 -> 126 -> 115 -> 703 -> 555 -> 698 -> 76 -> 387 -> 510 -> 541 -> 830 -> 692 -> 387 -> 662 -> 451 -> 411 -> 622 -> 654 -> 229 -> 786 -> 196 -> 753 -> 153 -> 742 -> 461 -> 401 -> 636 -> 313 -> 7 -> 502 -> 469 -> 377 -> 391 -> 425 -> 507 -> 558 -> 207 -> 794 -> 705 -> 424 -> 143 -> 538 -> 152 -> 673 -> 654 -> 721 -> 627 -> 737 -> 257 -> 146 -> 647 -> 175 -> 37 -> 176 -> 765 -> 651 -> 357
43416
SSTF
193 -> 191 -> 186 -> 177 -> 176 -> 175 -> 196 -> 202 -> 203 -> 207 -> 216 -> 229 -> 236 -> 245 -> 253 -> 254 -> 256 -> 257 -> 261 -> 265 -> 266 -> 279 -> 280 -> 291 -> 307 -> 313 -> 313 -> 317 -> 321 -> 327 -> 338 -> 339 -> 345 -> 354 -> 357 -> 362 -> 371 -> 371 -> 373 -> 377 -> 387 -> 387 -> 391 -> 399 -> 401 -> 411 -> 411 -> 421 -> 424 -> 425 -> 441 -> 449 -> 451 -> 451 -> 461 -> 469 -> 469 -> 480 -> 486 -> 488 -> 495 -> 502 -> 507 -> 510 -> 512 -> 513 -> 525 -> 531 -> 538 -> 541 -> 555 -> 558 -> 563 -> 583 -> 585 -> 588 -> 604 -> 605 -> 607 -> 616 -> 622 -> 623 -> 627 -> 636 -> 647 -> 650 -> 651 -> 653 -> 654 -> 654 -> 662 -> 666 -> 671 -> 673 -> 680 -> 684 -> 692 -> 698 -> 703 -> 705 -> 713 -> 721 -> 727 -> 733 -> 737 -> 742 -> 749 -> 753 -> 753 -> 754 -> 759 -> 765 -> 777 -> 786 -> 794 -> 809 -> 827 -> 830 -> 153 -> 152 -> 146 -> 145 -> 143 -> 140 -> 140 -> 126 -> 125 -> 124 -> 116 -> 115 -> 109 -> 96 -> 88 -> 87 -> 84 -> 82 -> 80 -> 76 -> 54 -> 50 -> 48 -> 47 -> 37 -> 35 -> 34 -> 33 -> 21 -> 9 -> 7 -> 2
1501
SCAN
193 -> 191 -> 186 -> 177 -> 176 -> 175 -> 153 -> 152 -> 146 -> 145 -> 143 -> 140 -> 126 -> 125 -> 124 -> 116 -> 115 -> 109 -> 96 -> 88 -> 87 -> 84 -> 82 -> 80 -> 76 -> 54 -> 50 -> 48 -> 47 -> 37 -> 35 -> 34 -> 33 -> 21 -> 9 -> 7 -> 2 -> 0 -> 196 -> 202 -> 203 -> 207 -> 216 -> 229 -> 236 -> 245 -> 253 -> 254 -> 256 -> 257 -> 261 -> 265 -> 266 -> 279 -> 280 -> 291 -> 307 -> 313 -> 317 -> 321 -> 327 -> 338 -> 339 -> 345 -> 354 -> 357 -> 362 -> 371 -> 373 -> 377 -> 387 -> 391 -> 399 -> 401 -> 411 -> 421 -> 424 -> 425 -> 441 -> 449 -> 451 -> 461 -> 469 -> 480 -> 486 -> 488 -> 495 -> 502 -> 507 -> 510 -> 512 -> 513 -> 525 -> 531 -> 538 -> 541 -> 555 -> 558 -> 563 -> 583 -> 585 -> 588 -> 604 -> 605 -> 607 -> 616 -> 622 -> 623 -> 627 -> 636 -> 647 -> 650 -> 651 -> 653 -> 654 -> 662 -> 666 -> 671 -> 673 -> 680 -> 684 -> 692 -> 698 -> 703 -> 705 -> 713 -> 721 -> 727 -> 733 -> 737 -> 742 -> 749 -> 753 -> 754 -> 759 -> 765 -> 777 -> 786 -> 794 -> 809 -> 827 -> 830
1023
C-SCAN
193 -> 191 -> 186 -> 177 -> 176 -> 175 -> 153 -> 152 -> 146 -> 145 -> 143 -> 140 -> 126 -> 125 -> 124 -> 116 -> 115 -> 109 -> 96 -> 88 -> 87 -> 84 -> 82 -> 80 -> 76 -> 54 -> 50 -> 48 -> 47 -> 37 -> 35 -> 34 -> 33 -> 21 -> 9 -> 7 -> 2 -> 0 -> 840 -> 830 -> 827 -> 809 -> 794 -> 786 -> 777 -> 765 -> 759 -> 754 -> 753 -> 749 -> 742 -> 737 -> 733 -> 727 -> 721 -> 713 -> 705 -> 703 -> 698 -> 692 -> 684 -> 680 -> 673 -> 671 -> 666 -> 662 -> 654 -> 653 -> 651 -> 650 -> 647 -> 636 -> 627 -> 623 -> 622 -> 616 -> 607 -> 605 -> 604 -> 588 -> 585 -> 583 -> 563 -> 558 -> 555 -> 541 -> 538 -> 531 -> 525 -> 513 -> 512 -> 510 -> 507 -> 502 -> 495 -> 488 -> 486 -> 480 -> 469 -> 461 -> 451 -> 449 -> 441 -> 425 -> 424 -> 421 -> 411 -> 401 -> 399 -> 391 -> 387 -> 377 -> 373 -> 371 -> 362 -> 357 -> 354 -> 345 -> 339 -> 338 -> 327 -> 321 -> 317 -> 313 -> 307 -> 291 -> 280 -> 279 -> 266 -> 265 -> 261 -> 257 -> 256 -> 254 -> 253 -> 245 -> 236 -> 229 -> 216 -> 207 -> 203 -> 202 -> 196
837
LOOK
193 -> 191 -> 186 -> 177 -> 176 -> 175 -> 153 -> 152 -> 146 -> 145 -> 143 -> 140 -> 126 -> 125 -> 124 -> 116 -> 115 -> 109 -> 96 -> 88 -> 87 -> 84 -> 82 -> 80 -> 76 -> 54 -> 50 -> 48 -> 47 -> 37 -> 35 -> 34 -> 33 -> 21 -> 9 -> 7 -> 2 -> 196 -> 202 -> 203 -> 207 -> 216 -> 229 -> 236 -> 245 -> 253 -> 254 -> 256 -> 257 -> 261 -> 265 -> 266 -> 279 -> 280 -> 291 -> 307 -> 313 -> 317 -> 321 -> 327 -> 338 -> 339 -> 345 -> 354 -> 357 -> 362 -> 371 -> 373 -> 377 -> 387 -> 391 -> 399 -> 401 -> 411 -> 421 -> 424 -> 425 -> 441 -> 449 -> 451 -> 461 -> 469 -> 480 -> 486 -> 488 -> 495 -> 502 -> 507 -> 510 -> 512 -> 513 -> 525 -> 531 -> 538 -> 541 -> 555 -> 558 -> 563 -> 583 -> 585 -> 588 -> 604 -> 605 -> 607 -> 616 -> 622 -> 623 -> 627 -> 636 -> 647 -> 650 -> 651 -> 653 -> 654 -> 662 -> 666 -> 671 -> 673 -> 680 -> 684 -> 692 -> 698 -> 703 -> 705 -> 713 -> 721 -> 727 -> 733 -> 737 -> 742 -> 749 -> 753 -> 754 -> 759 -> 765 -> 777 -> 786 -> 794 -> 809 -> 827 -> 830
1019
C-LOOK
193 -> 191 -> 186 -> 177 -> 176 -> 175 -> 153 -> 152 -> 146 -> 145 -> 143 -> 140 -> 126 -> 125 -> 124 -> 116 -> 115 -> 109 -> 96 -> 88 -> 87 -> 84 -> 82 -> 80 -> 76 -> 54 -> 50 -> 48 -> 47 -> 37 -> 35 -> 34 -> 33 -> 21 -> 9 -> 7 -> 2 -> 830 -> 827 -> 809 -> 794 -> 786 -> 777 -> 765 -> 759 -> 754 -> 753 -> 749 -> 742 -> 737 -> 733 -> 727 -> 721 -> 713 -> 705 -> 703 -> 698 -> 692 -> 684 -> 680 -> 673 -> 671 -> 666 -> 662 -> 654 -> 653 -> 651 -> 650 -> 647 -> 636 -> 627 -> 623 -> 622 -> 616 -> 607 -> 605 -> 604 -> 588 -> 585 -> 583 -> 563 -> 558 -> 555 -> 541 -> 538 -> 531 -> 525 -> 513 -> 512 -> 510 -> 507 -> 502 -> 495 -> 488 -> 486 -> 480 -> 469 -> 461 -> 451 -> 449 -> 441 -> 425 -> 424 -> 421 -> 411 -> 401 -> 399 -> 391 -> 387 -> 377 -> 373 -> 371 -> 362 -> 357 -> 354 -> 345 -> 339 -> 338 -> 327 -> 321 -> 317 -> 313 -> 307 -> 291 -> 280 -> 279 -> 266 -> 265 -> 261 -> 257 -> 256 -> 254 -> 253 -> 245 -> 236 -> 229 -> 216 -> 207 -> 203 -> 202 -> 196
825
